
# HG changeset patch
# User Philipp Kewisch <mozilla@kewis.ch>
# Date 1224668502 -7200
# Node ID b07444ad2c523d823c3abd4fbf83ff974c32e562
# Parent b22da03722e45cc2cbe641385a3fb21a188d9ee0
Fix bug 415509 - [Trunk] Only the last event of a day is selectable by mouse click in day/weekview. r=berend

--- a/calendar/base/content/calendar-multiday-view.xml	Wed Oct 22 11:38:54 2008 +0200
+++ b/calendar/base/content/calendar-multiday-view.xml	Wed Oct 22 11:41:42 2008 +0200
@@ -266,7 +266,7 @@
     <content>
       <xul:stack anonid="boxstack" flex="1" style="min-width: 1px; min-height: 1px">
         <xul:box anonid="bgbox" flex="1" style="min-width: 1px; min-height: 1px"/>
-        <xul:box xbl:inherits="context" anonid="topbox" flex="1" equalsize="always" style="min-width: 1px; min-height: 1px"/>
+        <xul:box xbl:inherits="context" anonid="topbox" flex="1" equalsize="always" style="min-width: 1px; min-height: 1px" mousethrough="always"/>
         <xul:box anonid="fgbox" flex="1" class="fgdragcontainer" style="min-width: 1px; min-height: 1px; overflow:hidden;">
           <xul:box anonid="fgdragspacer" style="display: inherit; overflow: hidden;">
             <xul:spacer flex="1"/>
@@ -1948,7 +1948,7 @@
      -  An individual event box, to be inserted into a column.
     -->
   <binding id="calendar-event-box" extends="chrome://calendar/content/calendar-view-core.xml#calendar-editable-item">
-    <content tooltip="itemTooltip">
+    <content tooltip="itemTooltip" mousethrough="never">
         <xul:box xbl:inherits="orient,width,height" flex="1">
           <xul:box anonid="event-container"
                    class="calendar-color-box"

